_specs/ecip-1019.md

```diff
@@ -3,7 +3,7 @@ lang: en
 ecip: 1019
 title: (Epoch Decay 10) Monetary Policy and Final Modification to the Ethereum Classic Emission Schedule
 author: Mike Boremi <ecip-1019@mikes.email>
-status: Draft
+status: Withdrawn
```

Comment:
```suggestion
status: Rejected
```

Reply:
Maybe this should actually be "Replaced"?

https://ecips.ethereumclassic.org/ECIPs/ecip-1017 and
https://ecips.ethereumclassic.org/ECIPs/ecip-1039

Or am I reading something wrong into this history, @meowsbits?